src/Squirrel.CommandLine/OSX/PackCommand.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using System.IO;
+
+namespace Squirrel.CommandLine.OSX
+{
+    public class PackCommand : BaseCommand
+    {
+        public Option<string> PackId { get; }
```

Comment:
Presumably this could share a base class with the windows pack/releasify command as there are a lot of shared arguments. Particularly these: PackId, PackVersion, PackDirectory, PackAuthors, PackTitle, IncludePdb, ReleaseNotes, NoDelta. 

SquirrelAwareExecutable is different in mac/windows. On windows this can be provided more than once, on mac it must be only specified once.

Reply:
It certainly could, I was a bit torn when I did this on if they should share a base class or not. I am happy to refactor this so that they are shared. I wasn't sure how much was similar by necessity vs coincidence.

Reply:
The specific arguments I mentioned are functionally equivalent in both OSX and Windows. They are both passed into `NugetConsole.CreatePackageFromMetadata`. I'm sure we could refactor parsing options & calling this function with some shared POCO to remove some boilerplate in the future. Happy to accept PR with or without this refactor but something to keep on the radar. The other similar options (eg. mainExe, icon etc) are different enough they should probably remain separate.